src/IssueViz.Security.UnitTests/HotspotsList/HotspotTableEntryTests.cs

```diff
@@ -85,6 +89,16 @@ public void TryGetValue_TextColumn_ReturnsHotspotMessage()
             value.Should().Be("test message");
         }
 
+        [TestMethod]
```

Comment:
Cosmetics: moved the test position so it would match the switch case position in code (I'm crazy, I know)